### PR TITLE
[Consignments] Update selected state of photos

### PR DESCRIPTION
**Pull request by mennenia**
Fixes https://github.com/artsy/consignments/issues/95

simulator screen shot - iphone 6 - 9 1 - 2017-11-30 at 15 35 55

### CODE REVIEW

File: src/lib/Components/Consignments/Components/ImageSelection.tsx [hunk 1]
```diff
@@ -64,19 +81,25 @@ const TakePhotoImage = (props: TakePhotoImageProps) =>
...
-  </TouchableHighlight>
+    <TouchableHighlight
+      onPress={() => props.onPressItem(props.data.image.uri)}
+      style={{
+        opacity: props.selected ? 0.5 : 1.0,
+      }}
+    >
+      <Image source={{ uri: props.data.image.uri }} style={{ height: 154, width: 154 }} />
+    </TouchableHighlight>
+    {props.selected ? <SelectedIcon /> : <View />}
```
Comment (mennenia) [inline src/lib/Components/Consignments/Components/ImageSelection.tsx]: can this be done nicer?

Comment (orta) [reply src/lib/Components/Consignments/Components/ImageSelection.tsx]: Barely: `props.selected || <SelectedIcon />`, kinda `props.selected ? <SelectedIcon /> : null` 

`null` is a better call than view, but whether the `||` is too far is subjective

Comment (mennenia) [reply src/lib/Components/Consignments/Components/ImageSelection.tsx]: `null` was the thing I was looking for. Duh! Will update :)

**Comment by mennenia** [comment]
@orta done :) anything else or merge on green?

**Comment by orta** [comment]
Nah, let's go

**Pull request merged**